packages/lib/src/components/Blik/components/BlikInput.tsx

```diff
@@ -56,6 +60,7 @@ function BlikInput(props: BlikInputProps) {
                     onInput: handleChangeFor('blikCode', 'input'),
                     onBlur: handleChangeFor('blikCode', 'blur'),
                     placeholder: '123456',
+                    inputMode: 'number',
```

Comment:
I think input mode should be `numeric` https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode

Reply:
And we should actually check if this produces the correct result because some browsers use the `pattern` attribute.

Reply:
@m1aw - can you elaborate on this comment about the `pattern` attribute?
Foe securedFields we only set `inputmode="numeric"`